File: src/MovieRental/Movie.cs
namespace MovieRental;

public class Movie
{
    public const int Regular = 0;
    public const int NewRelease = 1;
    public const int Childrens = 2;

    private string _title;
    private int _priceCode;

    public Movie(string title, int priceCode)
    {
        _title = title;
        _priceCode = priceCode;
    }

    public string Title => _title;

    public int PriceCode
    {
        get => _priceCode;
        set => _priceCode = value;
    }
}

File: src/movierental/movie.py
class Movie:
    REGULAR = 0
    NEW_RELEASE = 1
    CHILDRENS = 2

    def __init__(self, title, price_code):
        self._title = title
        self._price_code = price_code

    @property
    def title(self):
        return self._title

    @property
    def price_code(self):
        return self._price_code

    @price_code.setter
    def price_code(self, value):
        self._price_code = value
